db/schema.rb

```diff
@@ -0,0 +1,51 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_06_16_133815) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.integer "author_id"
+    t.integer "post_id"
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.integer "author_id"
+    t.integer "post_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.integer "author_id"
+    t.string "title"
+    t.text "text"
+    t.integer "comments_counter"
+    t.integer "likes_counter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
- [ ]  Great job on creating the migration for the Post model in your project! However, there is still room for improvement. One important aspect to consider is establishing the relationship between posts and users.
To achieve this, you should create a migration that adds a foreign key column in the Post model, referencing the author . This will ensure that each post is associated with a specific user

Reply:
- [ ] @muhammedb8826 great work implementing the changes requested, however you did it wrongly, you are suppose to generate a migration to crate the association instead of editing the schema file.